Assets/Scripts/Models/Furniture.cs

```diff
@@ -1105,7 +1104,7 @@ private bool DefaultIsValidPosition(Tile tile)
                 }
 
                 // Make sure tile is FLOOR
-                if (t2.Type != TileType.Floor && tileTypeBuildPermissions.Contains(t2.Type) == false)
+                if (t2.Type != TileType.Floor && tileTypeBuildPermissions.Contains(t2.Type.Type) == false)
```

Comment:
Notice that ```tileTypeBuildPermissions``` is now a HashSet of strings because when the prototypes are loaded we shouldn't assume that other prototypes are loaded and is worst with TileTypes since is not even in the PrototypeManager (i need to fix that :P), and not TileTypes as before. I don't think that comparing a string with a TileType will work.